Readme.md

```diff
@@ -46,15 +46,95 @@ If you'd like to build your own VFS for Git Windows installer:
 * Create a folder to clone into, e.g. `C:\Repos\VFSForGit`
 * Clone this repo into the `src` subfolder, e.g. `C:\Repos\VFSForGit\src`
 * Run `\src\Scripts\BuildGVFSForWindows.bat`
-* You can also build in Visual Studio by opening `src\GVFS.sln` (do not upgrade any projects) and building. However, the very first 
+* You can also build in Visual Studio by opening `src\GVFS.sln` (do not upgrade any projects) and building. However, the very first
 build will fail, and the second and subsequent builds will succeed. This is because the build requires a prebuild code generation step.
 For details, see the build script in the previous step.
 
 The installer can now be found at `C:\Repos\VFSForGit\BuildOutput\GVFS.Installer\bin\x64\[Debug|Release]\SetupGVFS.<version>.exe`
 
+## Building VFS for Git on Mac
+
+Note that VFS for Git on Mac is under active development.
+
+* Ensure you have Xcode installed, have accepted the terms of use, and have launched Xcode at least once.
+
+* Disable the "System Integrity Protection" (for loading unsigned Kexts) by booting into recovery mode (`[Win/⌘] + R` while booting).
+  Once booted into recovery mode open Utilities -> Terminal to launch a terminal. Enter:
+
+  ```
+  csrutil disable
+  ```
+  to disable SIP. Then click the Apple logo in the top left and restart.
+
+* Install [Visual Studio for Mac ](https://visualstudio.microsoft.com/vs/mac). (This will also install the `dotnet` CLI).
+
+* If you still do not have the `dotnet` cli `>= v2.1.300` installed [manually install it]. You can check what version you have with `dotnet --version`.(https://www.microsoft.com/net/download/dotnet-core/2.1)
+
+* You will need to manage and sign your own certificate. [Apple Developer Cert Docs](https://developer.apple.com/documentation/security/certificate_key_and_trust_services/certificates).

```

Comment:
This link isn't helpful to end users functionally need to just sign into Xcode, they don't need to learn about the APIs :)

On a fresh machine, I launched Xcode and opened the workspace. The build failed due to signing, clicking it took me to a page to sign in with my Apple ID which automatically fixed everything for me.

So I'd rephrase it like this:
If you're using Xcode for the first time, you may have to login to Xcode with your Apple ID to generate a codesigning certificate. 

or extra verbose:
If you're using Xcode for the first time, you may have to login to Xcode with your Apple ID to generate a codesigning certificate. You can do this by launching Xcode.app, opening the PrjFS.xcworkspace and trying to build. You can find the signing options in the General tab of the project's settings.

Reply:
I like your verbose option